concrete/src/Form/Service/Widget/DateTime.php

```diff
@@ -294,7 +296,7 @@ public function datetime($field, $value = null, $includeActivation = false, $cal
             $html .= <<<EOT
 <script type="text/javascript">
 $(function() {
-  $('#{$id}_dt_pub').datepicker({
+  $('#{$id}_dt_pub').datepicker($.extend({$datePickerOptionsAsJSON}, {
```

Comment:
I think we're missing a closing `)` for the newly open `(`. Am I wrong?

Reply:
I don't think you're right about this, did you see an error?